components/Tabs/Tabs.js

```diff
@@ -1,38 +1,42 @@
-
 class TabItem {
   constructor(element) {
     // attach dom element to object. Example in Tabs class
+    this.element = element;
   }
-
   select() {
     // should use classList
+    this.element.classList.add('Tabs__item-selected');
+    //this.element.classList.toggle('Tabs__item');
   }
-
   deselect() {
     // should use classList
+    this.element.classList.remove('Tabs__item-selected');
+    //this.element.classList.toggle('Tabs__item');
   }
 }
 
 class TabLink {
   constructor(element, parent) {
-    this.element;// attach dom element to object
-    this.tabs;// attach parent to object
-    this.tabItem;// assign this to the associated tab using the parent's "getTab" method by passing it the correct data
-    // reassign this.tabItem to be a new instance of TabItem, passing it this.tabItem
-    this.element.addEventListener('click', () => {
+    this.element = element;// attach dom element to object
```

Comment:
Make sure you are consistent with your comment styles you have some on the side others above.